modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/Actions.scss

```diff
@@ -14,4 +14,8 @@
 			border-radius: 0;
 		}
 	}
+
```

Comment:
Changed due to https://github.com/liferay-frontend/liferay-portal/pull/3765#discussion_r1369851785